controllers/hibp.js

```diff
@@ -36,6 +36,17 @@ async function notify (req, res) {
       throw new Error("Unrecognized breach: " + reqBreachName);
     }
   }
+  // If we don't have a logo of the breach, get the logo.
+  if (!fs.existsSync(`./public/img/logos/${breachAlert.LogoPath}`)) {
+    try {
+      await HIBP.downloadLogo(breachAlert.LogoPath);
+    } catch(err) {
+      // Set LogoPath to missing-logo-icon.png if download fails
+      // LogoPath will reset when the server restarts
+      log.error(`Unable to download logo for ${breachAlert.LogoPath}.`);
```

Comment:
I think this needs a string `op` as the first param and an object for the second param:

Something like:
```js
log.error("notify", {message: `Unable to download logo for ${breachAlert.LogoPath}`});
```